### ISSUE TITLE
Tracer A-Series

### ISSUE THREAD

**Issue by FritzBox360** [dangowrt/tracertools#1]
Hello, 

i am unable to get your tool to work with a Tracer 1210A. If i run ./tracerstats it takes a few seconds and i am back in my shell. No error or another output. The Traver A-Series should work with the MT-5 Display ( i have already tested it ) so your tool should also work i think? Is there a way to check it? 

Thank you for your work!

**Comment by dangowrt** [comment]
You are right, but I don't know this device. I initially made the tool by MitM sniffing the serial communication of the MT-5 display with the Tracer device by using 2x FTDI FT232 USB to serial 3.3V converters. You can build yourself a similar setup and figure out what's going on. Also, I noticed that the versions of the Tracer RN I encountered don't actually need the pwl start up sequence found the datasheet and other tools. Maybe this could be the reason for stuff not to work, because maybe we need some delay for the controller to come up after the pwl startup of something like that...
